web/src/games/schafkopf/components/GameBoard.tsx

```diff
@@ -104,7 +104,7 @@ export function Board(props: {
     return (
       <Trick
         trick={trick ? trick.cards : []}
-        leaderPos={trick ? +trick.leader.id : 0}
+        leaderPos={trick && trick.leader ? +trick.leader.id : 0}
```

Comment:
Before this PR, this line could lead to an error message `trick.leader is undefined`.

Reply:
Thanks for pointing out :)